docs/apache-airflow-providers-amazon/executors/ecs-executor.rst

```diff
@@ -125,51 +125,22 @@ integration, allowing you to interact with AWS services within your
 Airflow environment. It also includes options to load DAGs (Directed
 Acyclic Graphs) from either an S3 bucket or a local folder.
 
-Base Image
-~~~~~~~~~~
-
-The Docker image is built upon the ``apache/airflow:latest`` image. See
-`here <https://hub.docker.com/r/apache/airflow>`__ for more information
-about the image.
-
-Important note: The Airflow and python versions in this image must align
-with the Airflow and python versions on the host/container which is
-running the Airflow scheduler process (which in turn runs the executor).
-The Airflow version of the image can be verified by running the
-container locally with the following command:
-
-.. code-block:: bash
-
-   docker run <image_name> version
-
-Similarly, the python version of the image can be verified the following
-command:
-
-.. code-block:: bash
-
-   docker run <image_name> python --version
-
-Ensure that these versions match the versions on the host/container
-which is running the Airflow scheduler process (and thus, the ECS
-executor.) Apache Airflow images with specific python versions can be
-downloaded from the Dockerhub registry, and filtering tags by the
-`python
-version <https://hub.docker.com/r/apache/airflow/tags?page=1&name=3.8>`__.
-For example, the tag ``latest-python3.8`` specifies that the image will
-have python 3.8 installed.
+Download this image to use for the docker build commands below or create
+your own image if you prefer.
 
 Prerequisites
 ~~~~~~~~~~~~~
 
 Docker must be installed on your system. Instructions for installing
 Docker can be found `here <https://docs.docker.com/get-docker/>`__.
 
-AWS Credentials
-~~~~~~~~~~~~~~~
+Building an Image
```

Comment:
Changing this heading makes it more discoverable. This is where we start showing you how to build the image, so I updated the title to reflect that. It is also about credentials, but I think that's maybe the lesser important thing to call out in the heading.